<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>361</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="LayoutNumber2">
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButtonRemoveAll">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButtonSelectAll">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QTreeWidget" name="TreeWidget">
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>101</x>
     <y>55</y>
     <width>181</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="LayoutQuestion">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::FieldsStayAtSizeHint</enum>
    </property>
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QRadioButton" name="radioButtonStandartUpdate">
       <property name="text">
        <string>RadioButton</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButtonCustomUpdate">
       <property name="text">
        <string>RadioButton</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButtonExitFromUpdate">
       <property name="text">
        <string>RadioButton</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="LayoutGeneralFormButtons">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QPushButton" name="backButton">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
      <widget class="QPushButton" name="NextButton">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
      <widget class="QPushButton" name="CanselButton">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>321</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="LayoutProgresBar">
    <item>
     <widget class="QLabel" name="ProgressLabel">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="ProgressBar">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>NextButton</tabstop>
  <tabstop>CanselButton</tabstop>
  <tabstop>backButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
